docs/v3.x/getting-started/deployment.md

```diff
@@ -114,7 +114,8 @@ Additional guides for optional software additions that compliment or improve the
 
 To provide the best possible environment for Strapi there are a few requirements, these apply in both a development (local) as well as a staging and production workflow.
 
-- Node LTS (v10 or V12) **Note that Node v14 is not currently supported**
+- Node LTS (v12 or V14) **Note that Node "odd" releases will never be supported (v13/v15)**
```

Comment:
"Note that odd-number releases of Node will never be supported (e.g. v13, v15)."